DataFormats/SiStripCommon/src/SiStripEventSummary.cc

```diff
@@ -213,12 +213,11 @@ void SiStripEventSummary::commissioningInfo( const uint32_t& daq1,
   } else if ( runType_ == sistrip::PHYSICS_ZS ) { 
   } else if ( runType_ == sistrip::PEDESTALS ) { 
   } else if ( runType_ == sistrip::DAQ_SCOPE_MODE ) { 
-  } else if ( runType_ == sistrip::CALIBRATION ) { 
-  } else if ( runType_ == sistrip::CALIBRATION_DECO ) { 
+  } else if ( runType_ == sistrip::CALIBRATION_DECO or runType_ == sistrip::CALIBRATION_DECO) { 
```

Comment:
Logical OR with itself?

Reply:
Sorry, that was a clear typo .. updating now the PR